variables.tf

```diff
@@ -0,0 +1,87 @@
+variable "es_endpoint" {}
```

Comment:
add `description` to all variables.
suggest also adding `type` and `default` (if needed) to all vars
https://github.com/cloudposse/terraform-aws-eks-cluster/blob/master/variables.tf

for consistency, and for auto-generated README